### PR TITLE
v0.5.x into v0.6.x

### PR DESCRIPTION
**Pull request by benjaoming**
## Summary

Routine stuff, but fixes conflicts (introduced knowingly)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,11 +65,10 @@ docs: clean-docs
...
 
-release: clean assets
-	python setup.py sdist upload
-	python setup.py bdist_wheel upload
-	@echo ""
-	@echo "Now run something like twine -s dist/* to upload all results in the dist/ folder"
+release:
+	ls -l dist/
+	read "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
+	twine upload -s dist/*
```
Comment (indirectlylit) [inline Makefile:71]: cc @aronasorman just want to make sure you're aware of these changes in case it affects buildkite in any way

Comment (aronasorman) [reply Makefile:71]: Thanks, don't think it should affect buildkite -- we don't upload anything through there right now.

Comment (benjaoming) [reply Makefile:71]: I think releases on PyPI will continue to be a manual thing

Comment (benjaoming) [reply Makefile:71]: `twine` is the command that releases stuff to PyPI and `-s` means to PGP sign the release.

**Pull request merged**